coderd/database/migrate.go

```diff
@@ -36,8 +46,19 @@ func migrateSetup(db *sql.DB) (source.Driver, *migrate.Migrate, error) {
 }
 
 // MigrateUp runs SQL migrations to ensure the database schema is up-to-date.
-func MigrateUp(db *sql.DB) error {
+func MigrateUp(db *sql.DB) (retErr error) {
```

Comment:
Tip (non-blocking): It would be OK to call this `err`, at the time `defer func` runs, the value of `err` would be that of the `return` statement.